.github/workflows/build-test.yml

```diff
@@ -42,3 +42,55 @@ jobs:
           files: ./lcov.info
           token: ${{ secrets.CODECOV_TOKEN }} 
           verbose: true
+
+  publish:
+    name: Publish @leapfrogtechnology/async-store package
+    needs: build
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.RIBBY_GITHUB_TOKEN }}
+
+      - name: Set git user
+        run: |
+          git config user.name "Ribby"
+          git config user.email "bot@lftechnology.com"
+
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
+
+      - name: Install gems
+        run: |
+          gem install github_changelog_generator
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'yarn'
+
+      - name: Install node dependencies
+        run: yarn
+
+      - name: Install hub
+        run: |
+          sudo apt update
+          sudo apt install hub
+          hub --version
+
+      - name: Run release script
+        env:
+          GITHUB_TOKEN: ${{ secrets.RIBBY_GITHUB_TOKEN }}
```

Comment:
Are the token available with same name everywhere @mesaugat or else will update it

Reply:
I can keep it with the same name.

Reply:
cool